Bio/KDTree/KDTreemodule.c

```diff
@@ -1,3 +1,13 @@
+/* Copyright 2008-2019 by Michiel de Hoon.  All rights reserved.
```

Comment:
Michiel's most recent change was 2018, not 2019.

Reply:
I will edit it.